setup.py

```diff
@@ -131,24 +117,34 @@ def pip_install(package_name):
     cmdclass={
         'build_py': build_py,
         'develop': develop,
+        # 'build_ext': build_ext
+    },
+    install_requires=requirements,
+    python_requires='>=3.6.0, <3.9',
+    entry_points={
+        'console_scripts': [
+            'tts=TTS.bin.synthesize:main',
+            'tts-server = TTS.server.server:main'
+        ]
     },
-    install_requires=requirements['install_requires'],
-    python_requires='>=3.6.0',
     classifiers=[
         "Programming Language :: Python",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
This should be also tested in CI than.

Reply:
I removed python 3.9 since Tensorflow does not support 3.9 yet and it fails the TF interface we provide for Tacotron models.

Reply:
fair.